docs/observability/reliability-management-slo/dashboards.md

```diff
@@ -62,86 +67,110 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
+The SLO lookup table is a fully managed [lookup table](/docs/search/lookup-tables/create-lookup-table/#introduction-to-lookup-tables), that contains the latest definitions of all your SLOs. It can be use to enrich the data in your `sumologic_slo_output` view to build custom dashboards.
 
-There are two ways to use it:
+The SLO lookup table resides under a fixed path, `sumo://content/slos`. You can list all your SLOs using the log query `cat sumo://content/slos`.
 
-* To join the results of your SLO precomputed data from `_view=sumologic_slo_output` with your metadata contained in the internal lookup table based on the joining key (`sloId`, `sloVersion`):
+
+To join the results of your SLO precomputed data from `_view=sumologic_slo_output` with the metadata contained in the SLO lookup table, you can use the following query:
   ```sql
   _view=sumologic_slo_output
   | lookup * from sumo://content/slos on sloId, sloVersion
   ```
-* To enlist the contents of the lookup table:
-  ```sql
-  cat sumo://content/slos
-  ```
 
-As an example, say you had a SLO [dashboard](/docs/dashboards) and wanted to see error budget burndown from several of your apps and services combined.<br/><img src={useBaseUrl('img/observability/percent-error-remain.png')} alt="percent-error-remain" width="450"/>
+### Custom Dashboard Examples
 
-You would need to create a custom graphic that combines multiple SLOs from multiple services:
+#### Error Budget Remaining for all SLOs
 
-1. Go to **Manage Data** > **Monitoring** > **SLO**.
-1. Click on any SLO line item.
-1. Hover over the **Percentage budget remaining** panel, then click the three-dot icon > **Open in Log Search**.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
-1. In the search field, enter the following snippet. This will join data from multiple sources for your lookup table.
-  ```sql
-  _view=sumologic_slo_output
-  | lookup * from sumo://content/slos on sloId, sloVersion
-  | where !isBlank (sloname) and slofolderpath matches "*"
-  | concat (sloname, " (", sloId, ")") as sloUniqueName
-  | sum (goodCount) as goodEvents, sum(totalCount) as totalEvents, last (compliancetarget) as target, last(slofolderpath) as sloPath, last(sliwindowsize) as sliwindowsize, last(slievaluationtype) as evaluationType by sloUniqueName
-  | totalEvents - goodEvents as badEvents
-  | if (evaluationType = "Window", queryTimeRange() / 1000 / sliwindowsize, totalEvents) as denominator
-  | 100 * (1 - badEvents / denominator) as sli
-  | 100 * (sli - target) / (100 - target) as budgetRemaining
-  | fields sloUniqueName, budgetRemaining
-  ```
-1. Click **Add to Dashboard**.<br/><img src={useBaseUrl('img/observability/add-to-dashboard.png')} alt="add-to-dashboard" width="200"/>
+Say you want a bird's eye view into the health of your SLOs. A honeycomb visualization on the error budget remaining percentage is a nice way to achieve that.
```

Comment:
While bird's-eye view is correct, I'd change it as not all non-English speakers might know this phrase.

```suggestion
Say you want a high-level overview into the health of your SLOs. A honeycomb visualization on the error budget remaining percentage is a nice way to achieve that.
```